includes/table/class-following.php

```diff
@@ -105,7 +108,34 @@ public function process_action() {
 					}
 				}
 				break;
+			case 'follow':
+				$redirect_to = \remove_query_arg( array( 's' ), $redirect_to );
 
+				if ( ! isset( $_REQUEST['activitypub-profile'], $_REQUEST['_wpnonce'] ) ) {
+					return;
+				}
+
+				$nonce = \sanitize_text_field( \wp_unslash( $_REQUEST['_wpnonce'] ) );
+				if ( ! \wp_verify_nonce( $nonce, 'activitypub-follow-nonce' ) ) {
+					return;
+				}
+
+				$profile = \sanitize_text_field( \wp_unslash( $_REQUEST['activitypub-profile'] ) );
+
+				$post = Actors::fetch_remote_by_uri( $profile );
+
+				if ( ! \is_wp_error( $post ) ) {
+					$result = Following_Collection::follow( $post, $this->user_id );
+					if ( \is_wp_error( $result ) ) {
+						\add_settings_error( 'activitypub', 'followed', $result->get_error_message(), 'error' );
+					} else {
+						\add_settings_error( 'activitypub', 'followed', \__( 'Account followed.', 'activitypub' ), 'success' );
+					}
+				} else {
+					\add_settings_error( 'activitypub', 'followed', $post->get_error_message(), 'error' );
+				}
```

Comment:
```suggestion
				if ( \is_wp_error( $post ) ) {
					\add_settings_error( 'activitypub', 'followed', $post->get_error_message() );
					break;
				}

				$result = Following_Collection::follow( $post, $this->user_id );
				if ( \is_wp_error( $result ) ) {
					\add_settings_error( 'activitypub', 'followed', $result->get_error_message() );
				} else {
					\add_settings_error( 'activitypub', 'followed', \__( 'Account followed.', 'activitypub' ), 'success' );
				}
```
Not sure if this would make it a bit easier to read?